packages/react-core/src/demos/DateTimePicker.md

```diff
@@ -0,0 +1,104 @@
+---
+id: Date and time picker
+section: demos
+beta: true
+---
+
+import OutlinedCalendarAltIcon from '@patternfly/react-icons/dist/esm/icons/outlined-calendar-alt-icon';
+import OutlinedClockIcon from '@patternfly/react-icons/dist/esm/icons/outlined-clock-icon';
+
+## Demos
+
+In this section, we demonstrate how to use a [CalendarMonth](/components/calendar-month) and [Dropdown](/components/dropdown) to compose a simple DateTimePicker.
```

Comment:
@abigaeljamie do you mind giving feedback on the copy here.

Reply:
```suggestion
In this demo, learn how to use a calendar month component and a dropdown component to compose a simple date time picker. The input is read only, meaning that the following example does not cover pasting a copied date and time. Also, you can pick the time only from the predefined options and cannot change it in the input.
``` 

I tweaked it a little bit to follow some consistency standards (writing component names in lowercase) and to use the active voice to make it a bit easier for readers to follow. I also edited the kick-off a tad from "In this section, we demonstrate how..." to "In this demo, learn how to..."  

After reading the other demos on PatternFly, I'm identifying areas we can edit the content so that each demo follows the same structure for consistency. @mcarrano , I'm thinking this is something Allison and Katie can work on when they start if you think it would be a good idea to get some content editing going for our demos so that they all have a unified structure and voice 🙂

Reply:
That would be great @abigaeljamie . I suggest opening a new issue and attaching this feedback. @nicolethoen @tlabaj  can you advise on the best way to accomplish these updates as I know this test all lives in the React and/or Core repos.